release_creation/bundle/requirements/v1.7.pre.requirements.txt

```diff
@@ -0,0 +1,14 @@
+dbt-core~=1.7.0b1 --no-binary dbt-postgres
+dbt-snowflake~=1.7.0b1
```

Comment:
can we make these pull from 1.7.latest? similar to https://github.com/dbt-labs/dbt-core-bundles/blob/main/release_creation/bundle/requirements/v1.2.pre.requirements.txt

Reply:
Sure was curious if that will work if there is no 1.7.latest branch? this is a beta so all being done off main.